addon/components/paper-dialog-content.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import PaperDialogInner from './paper-dialog-inner';
+
+export default Ember.Component.extend({
+  tagName: 'md-dialog-content',
+  classNames: ['md-dialog-content'],
+
+  dialogInnerComponent: Ember.computed(function () {
+    return this.nearestOfType(PaperDialogInner);
+  }),
+  imagesLoaded () {
```

Comment:
this should be triggering jscs errors.

Run `jscs -c .jscsrc app addon` to see style errors.